pkg/orchestrator/instance.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2018 Pressinfra SRL
+Copyright 2014 Outbrain Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package orchestrator
+
+import (
+	"time"
+)
+
+// Note that this code is copied from:
+// https://github.com/github/orchestrator/blob/master/go/inst/instance.go We
+// need just the types that's way is copied instead of imported.
```

Comment:
`way` -> `why` and also please move the comment above the package declaration, so it shows up as package doc.